method/index.js

```diff
@@ -1,19 +1,52 @@
+const Crut = require('ssb-crut')
 const Init = require('./group/init')
 const AddMember = require('./group/add-member')
-const CreateLink = require('./link/create')
 const FindByGroupByFeedId = require('./link/find-group-by-feedid')
 const Application = require('./application')
+const FeedGroupLink = require('../spec/link/feed-group')
+const GroupSubgroupLink = require('../spec/link/group-subgroup')
 
 module.exports = function Method (ssb, keystore, state) {
   const application = Application(ssb)
+  const linkFeedCrut = new Crut(ssb, FeedGroupLink)
+  const linkSubgroupCrut = new Crut(ssb, GroupSubgroupLink)
```

Comment:
probably follow the pattern `Application` has, where it just takes ssb, and you don't need to `new Crut` here

Reply:
so yeah I think I'm saying "yes I reckon you can move both *link* thinks out into one `./link.sj` file"
But also doesn't matter that much